packages/zowe-explorer/CHANGELOG.md

```diff
@@ -4,13 +4,6 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ## TBD Release
 
-### Bug fixes
-
-- BugFix: Fixed an issue where an invalid token in a base configuration profile will take precedence over a plaintext user/password in a service configuration profile. [#3575](https://github.com/zowe/zowe-explorer-vscode/pull/3575)
-- BugFix: Fixed an issue where profiles that were not utilizing tokens were improperly running `checkJwtForProfile()` function within the `ZoweTreeProvider`. [#3575](https://github.com/zowe/zowe-explorer-vscode/pull/3575)
-- Fixed an issue where copy-and-pasting an empty PDS within the same LPAR would result in an error. [#3541](https://github.com/zowe/zowe-explorer-vscode/issues/3541)
-- Added Time Started, Time Ended, and Time Submitted job properties to the Jobs table view. [#3055](https://github.com/zowe/zowe-explorer-vscode/issues/3055)
```

Comment:
This was just a minor cleanup item.
- https://github.com/zowe/zowe-explorer-vscode/pull/3575#discussion_r2052963701

Reply:
Sorry I may be missing something regarding this comment 😅 is there an action I need to take here?

Reply:
Nope, nothing here... just clarifying why I did that as part of the branch-merge commit 😋